syscall/syscall.c

```diff
@@ -71,6 +71,17 @@ OE_WEAK OE_DEFINE_SYSCALL1(SYS_chdir)
     return oe_chdir(path);
 }
 
+// TODO: Implement this correctly
```

Comment:
We should not have TODO comments. Can you expand this in more detail (add a reference to a github issue is if necessary).

Reply:
I have added a subtask to correctly implement and test this syscall - #3933 This will be resolved as a part of MUSL upgrade.

Reply:
Can you remove the TODO comments and rephrase it (with the url to the github issue)?